cmd/state-remote-installer/main.go

```diff
@@ -193,12 +196,17 @@ func execute(out output.Outputer, prompt prompt.Prompter, cfg *config.Instance,
 	env := []string{
 		constants.InstallerNoSubshell + "=true",
 	}
-	_, _, err = exeutils.ExecuteAndPipeStd(filepath.Join(tmpDir, constants.StateInstallerCmd+exeutils.Extension), args, env)
+	_, cmd, err := exeutils.ExecuteAndPipeStd(filepath.Join(tmpDir, constants.StateInstallerCmd+exeutils.Extension), args, env)
 	if err != nil {
+		if cmd != nil && cmd.ProcessState.Sys().(syscall.WaitStatus).Exited() {
```

Comment:
Isn't there a possibility of a panic calling `.Exited()` if the type check failed?

Reply:
I thought so too, but this is what Go does:

https://github.com/golang/go/blob/master/src/os/exec_posix.go#L102